kalite/topic_tools/__init__.py

```diff
@@ -67,6 +78,20 @@ def get_topic_tree(force=False, annotate=False, channel=None, language=None, par
         TOPICS = {}
     if TOPICS.get(channel) is None:
         TOPICS[channel] = {}
+
+    if settings.DO_NOT_RELOAD_CONTENT_CACHE_AT_STARTUP and not force:
```

Comment:
I know what you mean, but the tagged issue was about writing caches in user data. Moving around that block of code had nothing to do with that, and it looks like it changes the behavior in the way that I noted. If you don't wanna clean up code, then I say resist the temptation to make unrelated changes!